docs(docusaurus): fix meta tags & add descriptions

ArnavK-09 opened the pull request:
###### /claim #21

ArnavK-09 committed:
docs(docusaurus): comment out unused Twitter metadata tags

The Twitter title and description metadata tags were commented out as they are currently unused. This change helps keep the configuration clean and focused on active metadata.

ArnavK-09 commented:
w8

ArnavK-09 committed:
docs: add descriptions to various documentation files

This commit adds `description` fields to multiple documentation files to improve clarity and provide more context for users. The descriptions explain the purpose and functionality of various elements, footprints, and guides in the tscircuit ecosystem.

ArnavK-09 committed:
chore: update logo image reference in docusaurus config

The logo image has been updated from an SVG to a PNG format. This change ensures consistency with the new logo file added to the static directory.

ArnavK-09 committed:
temp add prod url, might break dev but when pushed to prod will work

seveibar commented on docusaurus.config.ts:
```diff
@@ -62,15 +62,15 @@ const config: Config = {
       // LinkedIn & Open Graph Meta Tags
       { property: "og:type", content: "website" },
-      { property: "og:title", content: "tscircuit Docs" },
+      { property: "og:title", content: "Tscircuit Docs" },
```
```suggestion
      { property: "og:title", content: "tscircuit Docs" },
```

seveibar committed:
Update docusaurus.config.ts

seveibar commented on docusaurus.config.ts:
```diff
@@ -62,15 +62,15 @@ const config: Config = {
       // LinkedIn & Open Graph Meta Tags
       { property: "og:type", content: "website" },
       { property: "og:title", content: "tscircuit Docs" },
       { property: "og:description", content: "Create Electronics with TypeScript and React" },
```
Should we remove this?

seveibar commented on docs/building-electronics/ordering-prototypes.mdx:
```diff
@@ -1,6 +1,7 @@
 ---
 title: Ordering Prototypes
 sidebar_position: 4
+description: 'After [you''ve designed your device](./designing-electronics-from-scratch.md),'
```
❌

seveibar commented on docs/building-electronics/designing-electronics-from-scratch.md:
```diff
@@ -1,6 +1,7 @@
 ---
 title: Designing Electronics from Scratch
 sidebar_position: 2
+description: Typically electronics engineers go through the following steps when designing
```
❌

seveibar commented on docs/web-apis/autorouting-api.mdx:
```diff
@@ -1,5 +1,6 @@
 ---
 title: Autorouting API
+description: tscircuit allows heavy customization of the autorouting process. In tscircuit
```
Cut off

seveibar commented on docs/intro/quickstart-cli.md:
```diff
@@ -1,6 +1,7 @@
 ---
 title: Quickstart CLI
 sidebar_position: 4
+description: 'The command line is the best way to use tscircuit. Using the CLI, you can just'
```
Incomplete

seveibar reviewed (changes requested):
Can we double check descriptions for completeness.

Future issue: lint to make sure description is always present

ArnavK-09 committed:
docs: update descriptions in documentation files

Update the descriptions in several documentation files to provide more detailed and accurate information about the features and workflows. This improves clarity and helps users better understand the processes and tools available.

ArnavK-09 committed:
refactor(docusaurus): comment out og:title and og:description tags

The og:title and og:description meta tags have been commented out to temporarily remove them from the website's Open Graph metadata. This change was made to test the impact of these tags on social media sharing behavior.

seveibar commented on docs/advanced/math-utils.mdx:
```diff
@@ -1,5 +1,6 @@
 ---
 title: Math Utils
+description: The `@tscircuit/math-utils` package provides a set of utilities that are
```
I dont think descriptions should have incomplete descriptions, even if it goes over the limit

seveibar commented on docs/building-electronics/what-are-electronics-made-of.mdx:
```diff
@@ -1,6 +1,7 @@
 ---
 title: What are electronics made of?
 sidebar_position: 1
+description: 'In this tutorial, we''re going to be talking about the different elements that'
```
Another incomplete sentence

seveibar reviewed (commented):
Make sure to check all descriptions. If we dont do descriptions right the first time nobody will know or remember to fix it

ArnavK-09 committed:
docs: update descriptions in math-utils and what-are-electronics-made-of

Improve clarity and provide additional context for the `@tscircuit/math-utils` package and the elements of a Printed Circuit Board (PCB) in the respective documentation files.

ArnavK-09 committed:
Update led.mdx

ArnavK-09 committed:
Update net.mdx

ArnavK-09 committed:
Update potentiometer.mdx

ArnavK-09 committed:
Update pushbutton.mdx

ArnavK-09 committed:
Update resistor.mdx

ArnavK-09 committed:
Update subcircuit.mdx

ArnavK-09 committed:
Update switch.mdx

ArnavK-09 committed:
Update via.mdx

ArnavK-09 committed:
Update footprinter-strings.mdx

ArnavK-09 committed:
Update essential-elements.mdx

ArnavK-09 committed:
Update importing-from-circuit-json.mdx

ArnavK-09 committed:
Update layout-properties.mdx

ArnavK-09 committed:
Update platform-configuration.md

ArnavK-09 committed:
Update running-tscircuit-inside-an-iframe.mdx

ArnavK-09 committed:
Update simple-route-json.mdx

ArnavK-09 committed:
Update pinheader.mdx

ArnavK-09 committed:
Update understanding-fabrication-files.md
